CMakeLists.txt

```diff
@@ -184,21 +187,24 @@ if("${gRPC_PROTOBUF_PROVIDER}" STREQUAL "module")
       message(WARNING "gRPC_PROTOBUF_PROVIDER is \"module\" but PROTOBUF_ROOT_DIR is wrong")
   endif()
 elseif("${gRPC_PROTOBUF_PROVIDER}" STREQUAL "package")
-  find_package(protobuf CONFIG)
-  if(protobuf_FOUND)
+  find_package(Protobuf ${gRPC_PROTOBUF_PROVIDER_TYPE})
+  if(PROTOBUF_FOUND)
```

Comment:
Uppercase variables will be defined only from `FindProtobuf` module for backward compatibility with he old implementation (CMake less 3.6 version). This wariable will not be defined for `Config-file Package`.